src/swipe/swipe.service.ts

```diff
@@ -0,0 +1,25 @@
+import { UserSwipe, UserSwipeType } from './entities';
+import { UserService } from '../user';
+
+export const SwipeService = new (class {
+    async changeUser(
```

Comment:
What?

Why is there an entire swipe module? It should be only in #59
I don't understand. You probably merged something wrong.